### DISCUSSION TITLE
Map display for workout?

### DISCUSSION THREAD

**Discussion by Paksuhomo** [dvmarinoff/Auuki#196] [Ideas]
I kind of miss some true virtual rides where you could join structured workout (from example from Best Bike Split) and real world gpx so you could ride your outdoor routes inside with live mapping. 

Anyone else miss this kind of feature? Or do you guys only watch your numbers? 😋

**Comment by dvmarinoff** [comment]
Currently you can upload a real world gps file, but only in the .fit format. You will get 2D profile of the elevation instead of the workout.
I don't plan on supporting gpx files, but the the rest of the functionality can be improved. It's just that very few people care about this.

I typically watch on-demand sports (Cycling mostly) while using the turbo, so I prefer ERG mode and an automated workout, that can keep me in the correct Zone, while I focus on watching the Race.
